src/AppBundle/Repository/PostRepository.php

```diff
@@ -28,6 +28,14 @@
 class PostRepository extends EntityRepository
 {
     /**
+     * @return Post[]
+     */
+    public function findAll()
```

Comment:
I'm fine either way and had considered these options. I figured it would be easier to submit a PR and start that discussion if someone objected to overriding `findAll`. :)